airbyte-integrations/connectors/source-snowflake/source_snowflake/streams/util_streams.py

```diff
@@ -156,33 +154,25 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
         :return an iterable containing each record in the response
         """
         response_json = response.json()
+        current_time = None
+        offset = 0
         column_names_to_be_extracted_from_records = [self.CURRENT_TIME_COLUMN_NAME]
         index_of_columns_from_names = self.get_index_of_columns_from_names(response_json, column_names_to_be_extracted_from_records)
 
         current_time_column_name_index = index_of_columns_from_names[self.CURRENT_TIME_COLUMN_NAME]
+
         for record in response_json.get("data", []):
             try:
-                yield {'current_time': record[current_time_column_name_index]}
+                current_time = record[current_time_column_name_index]
             except Exception:
                 error_message = 'Unexpected error while reading record'
                 emit_airbyte_error_message(error_message)
 
+        if current_time and ' ' in current_time:
```

Comment:
Same comment about the usage of the literal comparisons as already mentioned.

Reply:
Done